src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/PrisonerIndexService.kt

```diff
@@ -285,12 +295,82 @@ class PrisonerIndexService(
       }
     }
 
-  open class IndexBuildException(val error: String) :
-    Exception("Unable to build index reason: $error")
+  @Async
+  fun doCompare() {
+    try {
+      val start = System.currentTimeMillis()
+      val (onlyInIndex, onlyInNomis) = compareIndex()
+      val end = System.currentTimeMillis()
+      telemetryClient.trackEvent(
+        "POSIndexReport",
```

Comment:
rename the event to start with `POS` like the other prisoner offender search events